anna.txt

```diff
@@ -1 +1,2 @@
-Hello from Caoimhe
+Anna Test
+Hello
```

Comment:
These lines could maybe be merged into one line? Saves 0.000000000001 bytes of storage space.